python/ray/tune/trainable.py

```diff
@@ -206,6 +206,16 @@ def train(self):
 
         return result
 
+    def evaluate(self):
```

Comment:
Why is this a separate method? Isn't the evaluate flag on train enough?

Reply:
This was proposed (to which you agreed here) https://groups.google.com/d/msg/ray-dev/S8cN4IKrdWo/liYwLA0-CgAJ

Reply:
Models should be able to be evaluated without updating - similar to https://keras.io/models/model/#evaluate